docs/modules/user-guide/partials/proc_tagging-and-pushing-a-registry-container-image.adoc

```diff
@@ -0,0 +1,24 @@
+[id="building-a-custom-devfile-registry_{context}"]
+= Tagging and pushing a registry container image
+
+[role="_abstract"]
+Use the `devfile index` container image to create a container in which you build, run, and deploy your application.
+
+.Procedure
+
+. Choose a container registry.
+. Consult the documentation for your container registry and log into the registry with the Docker CLI.
+. Push the devfile index container image to your chosen container registry:
```

Comment:
We have two different ways of formatting "devfile index container image" in this document ("`devfile index` container image" versus "devfile index container image"). Which one of these is accurate?

Reply:
removed the monospace, let me know if you think differently.